pkg/server/proto/server.proto

```diff
@@ -4666,6 +4680,23 @@ message RunPipelineResponse {
   map<string, Ref.PipelineStep> job_map = 3;
 }
 
+message ListPipelinesRequest {
+  // The project to list all pipelines on
+  Ref.Project project = 1;
+}
+
+message ListPipelinesResponse {
+  // All of the listed pipelines
+  repeated Pipeline pipelines = 1;
+}
+
+message ConfigSyncPipelineRequest {
+  // The project to sync all pipeline configs on
+  Ref.Project project = 1;
+}
+
+message ConfigSyncPipelineResponse {}
```

Comment:
Would be cool if we can get the IDs of the synced pipelines. Similar to `RunPipeline` if we can respond with like a `map<string, Ref.PipelineId> pipelines` that maps the pipeline name in the given project to the ID, that'd be cool.